receiver/kafkareceiver/azureresourcelogs_unmarshaler.go

```diff
@@ -0,0 +1,32 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package kafkareceiver // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/kafkareceiver"
+
+import (
+	"go.opentelemetry.io/collector/pdata/plog"
+	"go.uber.org/zap"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/azure"
+)
+
+type AzureResourceLogsUnmarshaler struct {
```

Comment:
This should not be exported 
```suggestion
type azureResourceLogsUnmarshaler struct {
```